lambda/lambda.mli

```diff
@@ -762,9 +762,31 @@ and lambda_event_kind =
   | Lev_function
   | Lev_pseudo
 
+(* Descriptor for a parameter that this module takes at runtime. *)
+type runtime_param_descr =
+  | Rp_argument_block of Global_module.t  (* The argument block of a module
+                                             compiled with [-as-argument-for] *)
+  | Rp_dependency of Global_module.t      (* A parameterised module (not itself a
+                                             parameter) that this module depends
+                                             on *)
```

Comment:
> `Rp_dependency x` is a runtime parameter that takes the module `x`, which is not completely instantiated; in particular, `x.hidden_args` is non-empty

Reply:
Good idea to mention the invariant, but it's actually not quite that (adding a “see `is_complete`” instead)